cdk.json

```diff
@@ -41,6 +41,17 @@
       "ACM_CERT_ARN": "arn:aws:acm:us-east-1:631692904429:certificate/0e9682f6-3ffa-46fb-9671-b6349f5164d6",
       "VPC_CIDR": "10.255.73.0/24"
     },
+    "uat": {
```

Comment:
You will need to add a section to deploy this uat context, https://github.com/Sage-Bionetworks/schematic-infra/blob/dev/.github/workflows/aws-deploy.yml#L66-L92

Reply:
Thanks. I just added a section of deploying the staging instance to the workflow.